werkzeug_encryptedcookie/__init__.py

```diff
@@ -66,6 +80,12 @@ def unserialize(cls, string, secret_key):
             except Exception:
                 pass
 
+        if cls.compress_cookie:
+            try:
+                string = cls.decompress(string)
+            except brotli.error:
```

Comment:
Brotli can successfully decompress random strings and return empty strings. Proof:

```python
import os, brotli
def test_b(): 
    s = os.urandom(89) 
    try: 
        brotli.decompress(s) 
    except brotli.error: 
        return 
    except: 
        pass 
    print('>>>', repr(s), repr(brotli.decompress(s))) 

%timeit test_b()
```

We need some kind of marker.

Reply:
Not sure if we should check for header or just check for `startswith` and then cur the `data`.
Maybe two checks for `separator` after `partition()` and check of result of decompression is enough. What do you think? @homm